components/browser/menu/src/main/res/layout/mozac_browser_menu.xml

```diff
@@ -12,8 +12,6 @@
 
         <androidx.recyclerview.widget.RecyclerView
             android:id="@+id/mozac_browser_menu_recyclerView"
-            android:paddingTop="@dimen/mozac_browser_menu_padding_vertical"
-            android:paddingBottom="@dimen/mozac_browser_menu_padding_vertical"
```

Comment:
You could override this dimension in Fenix. :)

Reply:
@pocmo is there a reason you think this should not be set for all of our menus? This helps menus even in reference browser and sample browser look truly centered (right now the top and bottom elements are not vertically centered due to padding).

Reply:
I think that is possible since:
* The material design specs define top and bottom padding for menus
* The UX design for Focus included top and bottom padding for the menu

How about a compromise: We keep the dimensions but set them to 0 by default. This makes 0 our standard but apps can still reconfigure the dimension.